main.py

```
@@ -0,0 +1,375 @@
+    state: str = (ev.get("state") or "").lower()
+
+    start_raw = ev.get("start")
+    end_raw = ev.get("end")
+    if not start_raw or not end_raw:
+        raise HTTPException(
+            status_code=422, detail="Event is missing start/end")
+
+    try:
+        dt_start = parse_iso_dt(start_raw)
+        dt_end = parse_iso_dt(end_raw)
+    except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)